src/core/financial.jl

```diff
@@ -48,8 +48,21 @@ function Financial(;
     microgrid_upgrade_cost_pct::Real = off_grid_flag ? 0.0 : 0.3,
     macrs_five_year::Array{Float64,1} = [0.2, 0.32, 0.192, 0.1152, 0.1152, 0.0576],  # IRS pub 946
     macrs_seven_year::Array{Float64,1} = [0.1429, 0.2449, 0.1749, 0.1249, 0.0893, 0.0892, 0.0893, 0.0446],
+    CO2_cost_per_tonne::Float64 = 51.0,
```

Comment:
should we replace all `tonne`s with `metric_ton`? Or is `tonne` commonly used for pollutant costs @adfarth?

Reply:
I think `tonne` explicitly equates to `metric ton` and is used often as a differentiator from `ton`. I therefore think it's fine to leave as `tonne`.